Assets/Scripts/Tags/NightLightTag.cs

```diff
@@ -3,4 +3,57 @@
 
 public class NightLightTag : MonoBehaviour
 {
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float _timeOn = 0.6f;
+	[SerializeField] float _fadeTime = 1.0f;
```

Comment:
The time it takes for the light to fade in. What would a more appropriate name be? `_fadeInTime`?